### PR TITLE
[Script] [heal-remedy] [Update] Fixes

### PR DESCRIPTION
**Pull request by Dartellum**
Removed errant comma, changed to use forward slashes for catches, and added a missing setting to two sections.

### CODE REVIEW

**Comment by rpherbig** [comment]
Changes look good to me ✔️

**Comment by Dartellum** [comment]
> Changes look good to me

Unfortunately, it still was not catching the line. I finally got it working but then ran into requiring an empty hand or variable passed to skip. I have the workaround in and testing.

File: heal-remedy.lic [hunk 1]
```diff
@@ -52,38 +52,38 @@ def wounds
   hurt = hurt.map { |raw| wound_grouping[raw] || raw }.uniq
   scar = hurt.map { |raw| scar_grouping[raw] || raw }.uniq
   if $debug_mode_hr
-    echo 'Wounds: '
+    DRC.message("Wounds: ")
```
Comment (rpherbig) [inline heal-remedy.lic:55]: Aren't a lot of these `message`s redundant? I.e. it echoes `hurt` on line 57 and then again on line 68. Same with `scar` on lines 64 and 76-77.

**Comment by rpherbig** [comment]
> The only echo I could not change and make work was line 51: `echo wounds if $debug_mode_hr`. I tried a few ways but could not get DRC.message to work with it. Suggestions?

It seems like this should be `DRC.message("wounds: #{wounds}") if $debug_mode_hr` - does that not work?

**Comment by Dartellum** [comment]
> > The only echo I could not change and make work was line 51: `echo wounds if $debug_mode_hr`. I tried a few ways but could not get DRC.message to work with it. Suggestions?
> 
> 
> 
> It seems like this should be `DRC.message("wounds: #{wounds}") if $debug_mode_hr` - does that not work?

No, that did not produce anything. I will play with it more tonight.

File: heal-remedy.lic (hunk 1 above)
Comment (Dartellum) [reply heal-remedy.lic:55]: The key changes for wounds and then for scars. May seem redundant but it was a sanity check to make sure everything was resolving properly through the different steps.

**Comment by Dartellum** [comment]
The output:
```
2021-08-04 19:46:14 -0400:[heal-remedy: {1=>[#<DRCH::Wound:0x00007fd7103a88f8 @body_part="neck", @severity=1, @bleeding_rate=nil, @is_internal=false, @is_scar=true>, #<DRCH::Wound:0x00007fd7103a86c8 @body_part="right arm", @severity=1, @bleeding_rate=nil, @is_internal=false, @is_scar=true>, #<DRCH::Wound:0x00007fd7103a8498 @body_part="right leg", @severity=1, @bleeding_rate=nil, @is_internal=false, @is_scar=true>, #<DRCH::Wound:0x00007fd7103a8268 @body_part="chest", @severity=1, @bleeding_rate=nil, @is_internal=false, @is_scar=true>, #<DRCH::Wound:0x00007fd7103a8038 @body_part="right eye", @severity=1, @bleeding_rate=nil, @is_internal=false, @is_scar=true>]}] From echo
2021-08-04 19:46:14 -0400:| Wounds: {1=>[#, #, #, #, #]} From DRC.message("wounds: #{wounds}") 
2021-08-04 19:46:14 -0400:| Wounds: 
2021-08-04 19:46:14 -0400:| key: neck
2021-08-04 19:46:14 -0400:| key: limbs
2021-08-04 19:46:14 -0400:| key: chest
2021-08-04 19:46:14 -0400:| key: eyes
2021-08-04 19:46:14 -0400:| Scars: 
2021-08-04 19:46:14 -0400:| key: face
2021-08-04 19:46:14 -0400:| key: limbs
2021-08-04 19:46:14 -0400:| key: body
2021-08-04 19:46:14 -0400:| key: neck
```
As you can see, `DRC.message("wounds: #{wounds}") if $debug_mode_hr` does not provide the output the `echo wounds  if $debug_mode_hr` does. Also, note it displays severity on this echo. The next output just shows the individual parts that will receive healing based on the severity setting (level). The two outputs help identify if it sees the wound and what severity it is and then to tell the user what it will heal.

**Comment by Dartellum** [comment]
Running through the updated logic for the wound display.

**Comment by wstampley** [comment]
> As you can see, `DRC.message("wounds: #{wounds}") if $debug_mode_hr` does not provide the output the `echo wounds if $debug_mode_hr` does

I think you can "wounds: #{wounds.to_yaml}" or something like that to get a pretty print of the DRCH wounds object.
From one of my scripts: `    DRC.message("wound-thief-debug: #{health.to_yaml}") if @debug`

**Comment by Dartellum** [comment]
> > As you can see, `DRC.message("wounds: #{wounds}") if $debug_mode_hr` does not provide the output the `echo wounds if $debug_mode_hr` does
> 
> I think you can "wounds: #{wounds.to_yaml}" or something like that to get a pretty print of the DRCH wounds object.
> From one of my scripts: ` DRC.message("wound-thief-debug: #{health.to_yaml}") if @debug`

I can give it a go. :smile:

**Comment by Dartellum** [comment]
I am also thinking about changing the debug logic from an arg passed in when running to a setting in your YAML.

**Comment by Dartellum** [comment]
> > > As you can see, `DRC.message("wounds: #{wounds}") if $debug_mode_hr` does not provide the output the `echo wounds if $debug_mode_hr` does
> > 
> > 
> > I think you can "wounds: #{wounds.to_yaml}" or something like that to get a pretty print of the DRCH wounds object.
> > From one of my scripts: ` DRC.message("wound-thief-debug: #{health.to_yaml}") if @debug`
> 
> I can give it a go.
Looks good:
```
2021-08-10 08:34:24 -0400:| wounds: ---
2021-08-10 08:34:24 -0400:1:
2021-08-10 08:34:24 -0400:- !ruby/object:DRCH::Wound
2021-08-10 08:34:24 -0400:  body_part: neck
2021-08-10 08:34:24 -0400:  severity: 1
2021-08-10 08:34:24 -0400:  bleeding_rate: 
2021-08-10 08:34:24 -0400:  is_internal: false
2021-08-10 08:34:24 -0400:  is_scar: false
2021-08-10 08:34:24 -0400:- !ruby/object:DRCH::Wound
2021-08-10 08:34:24 -0400:  body_part: right arm
2021-08-10 08:34:24 -0400:  severity: 1
2021-08-10 08:34:24 -0400:  bleeding_rate: 
2021-08-10 08:34:24 -0400:  is_internal: false
2021-08-10 08:34:24 -0400:  is_scar: false
2021-08-10 08:34:24 -0400:- !ruby/object:DRCH::Wound
2021-08-10 08:34:24 -0400:  body_part: right leg
2021-08-10 08:34:24 -0400:  severity: 1
2021-08-10 08:34:24 -0400:  bleeding_rate: 
2021-08-10 08:34:24 -0400:  is_internal: false
2021-08-10 08:34:24 -0400:  is_scar: false
```

I am doing some other tidying up now and testing this on another area.

**Comment by Dartellum** [comment]
This change is all ready to go. It has worked as expected for the last few weeks for me.

**Pull request merged**